lighthouse-core/audits/offline-start-url.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit.js');
+
+class OfflineStartUrl extends Audit {
+  /**
+   * @return {LH.Audit.Meta}
+   */
+  static get meta() {
+    return {
+      id: 'offline-start-url',
+      title: 'start_url responds with a 200 when offline',
```

Comment:
There's a text update on the `works-offline` audit. Something like "Current page... responds with a 200"

dunno if you want to do that here but we shouldn't drop it.

Reply:
> here's a text update on the `works-offline` audit. Something like "Current page... responds with a 200"
>
> dunno if you want to do that here but we shouldn't drop it.


The current proposed update is `"Current page responds with a 200 when offline"`. I modeled this after that...not sure what you mean by dropping it, though.

`Page at start_url responds with a 200 when offline`?